aten/src/ATen/native/quantized/cpu/qembeddingbag_prepack.cpp

```diff
@@ -77,7 +77,7 @@ c10::intrusive_ptr<EmbeddingPackedParamsBase> PackedEmbeddingBagWeight::prepack(
       channel_zero_points.data_ptr<float>() + embedding_rows);
 
   for (const auto i : c10::irange(embedding_rows)) {
-    weight_bias[i] = weight_zero_points[i] * weight_scales[i] * -1;
+    weight_bias[i] = weight_zero_points.at(i) * weight_scales.at(i) * -1;
```

Comment:
`at` may be slower and hurt loop optimisation (vectorisation). We rarely use `at` in intensive computing.